addons/payment_adyen/controllers/main.py

```diff
@@ -270,7 +276,7 @@ def adyen_notification(self):
                     notification_data['resultCode'] = 'Authorised'
                 elif event_code == 'CANCELLATION' and success:
                     notification_data['resultCode'] = 'Cancelled'
-                elif event_code == 'REFUND':
+                elif event_code in ['REFUND', 'CAPTURE']:
                     notification_data['resultCode'] = 'Authorised' if success else 'Error'
                 else:
```

Comment:
I didn't include it since, as I understand it, a successful `CAPTURE` is sent on these cases, but the real capture will happen later. 
But I could log it, that's true, that would make sense. As well as any failed cancellation (that's bothering me with the last changes we made, there isn't a log note for failed cancellation anymore).
I'll add both

Reply:
a successful `CAPTURE` is still sent shortly after the `CAPTURE_FAILED` ?
I didn't understand that after reading their doc, but if it's the case, it's probably less a problem then :thinking:

ANV nearly said it could eventually be another task for support & feedback of failed events ^^.

Reply:
`resolved`? Kinda